wrappers/services.go

```diff
@@ -421,6 +427,9 @@ RemainAfterExit={{.Remain}}
 BusName={{.App.BusName}}
 {{- end}}
 {{- if not .App.Sockets}}
+{{- if .KillMode}}
```

Comment:
This probably means that someone doing `systemctl stop snap.foo.bar.service` will get unexpected behavior.

Reply:
Yes, I can dig deeper but AFAICT there is no way to disentangle this :/